src/rejection_intake.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if (!defined('not_direct_access') && !empty($_POST['pre']) && !empty($_POST['post']) && !empty($_REQUEST['hunter_id_hash'])) {
+    define('not_direct_access', TRUE);
+    require_once "check-cors.php";
+    require_once "uuid.php";
+    require_once "check-ban.php";
+    require_once "check-time.php";
+    require_once "config.php";
+    require_once "db-connect.php";
+    require_once "send_response.php";
+    recordRejectionsInFile($_POST['pre'], $_POST['post']);
+    sendResponse('success', "Thanks for the hunt info!");
+}
+
+sendResponse('success', "Thanks for the hunt info!");
```

Comment:
in fact i would convert this whole thing into a guard clause

```suggestion
if (empty($_POST['pre']) || !empty($_POST['post']) || empty($_REQUEST['hunter_id_hash'])) {
    error_log("Missing pre, post, or hunter id hash");
    sendResponse('success', "Thanks for the hunt info!");
    die();
}

define('not_direct_access', TRUE);
require_once "check-cors.php";
require_once "uuid.php";
require_once "check-ban.php";
require_once "check-time.php";
require_once "config.php";
require_once "db-connect.php";
require_once "send_response.php";
recordRejectionsInFile($_POST['pre'], $_POST['post']);
sendResponse('success', "Thanks for the hunt info!");
```